ui/lib/core/addon/components/text-file.hbs

```diff
@@ -0,0 +1,98 @@
+{{#unless @hideTextAreaToggle}}
+  <div class="level is-mobile">
+    <div class="level-left">
+      <label class="is-label" data-test-text-file-label>
+        {{or @label "File"}}
+        {{#if @helpText}}
+          <InfoTooltip>
+            <span data-test-help-text>
+              {{@helpText}}
+            </span>
+          </InfoTooltip>
+        {{/if}}
+      </label>
+    </div>
+    <div class="level-right">
+      <div class="control is-flex">
+        <Input
+          data-test-text-toggle
+          id="text-area-toggle"
+          class="switch is-rounded is-success is-small"
+          @type="checkbox"
+          @checked={{this.showTextArea}}
+          {{on "change" (fn (mut this.showTextArea) (not this.showTextArea))}}
+        />
+        <label for="text-area-toggle">
+          Enter as text
+        </label>
+      </div>
+    </div>
+  </div>
+{{/unless}}
+<div class="field text-file box is-fullwidth is-marginless is-shadowless is-paddingless" data-test-component="text-file">
+  {{#if this.showTextArea}}
+    <div class="control has-icon-right">
+      <textarea
+        class="textarea {{if (and (not this.showValue) this.content) 'masked-font'}}"
```

Comment:
added check for `this.content` because the masked input defaults with values looking like there's content 

![text-file](https://user-images.githubusercontent.com/68122737/208484214-dd306dbd-3a4b-4d9d-8d72-cf5d6f61cca9.gif)